src/chapters/git/branches.rst

```diff
@@ -0,0 +1,45 @@
+Branches
+========
+
+.. index:: ! branch
+
+Branching in Git
+----------------
+
+So far this book has talked about Git's ability to store different versions of a code base.
+In the previous section, when a programmer checked the commit log, Git showed a record of changes over time.
+What if two programmers want to make different changes at the same time?
```

Comment:
Branches are to keep different "contexts" or "stories" of work separate. Git without branches handles two people working on the same file quite well. The git tower book has a short summary of it. https://www.git-tower.com/learn/git/ebook/en/command-line/branching-merging/branching-can-change-your-life#start